Sources/ReleaseSubscriptions/App.swift

```diff
@@ -41,3 +54,9 @@ struct App: AsyncParsableCommand {
         }
     }
 }
+
+extension Logger {
+    fileprivate static var app: Logger {
+        Logger(label: "io.github.ios-osushi.releasesubscriptions")
+    }
```

Comment:
Computed Property だと毎回生成されるから、Stored Property でシングルトンにしていいような気がする

```suggestion
    fileprivate static let app = Logger(label: "io.github.ios-osushi.releasesubscriptions")
```

Reply:
👍 
なぜか `protocol` と勘違いして「Stored Property にできないしなぁ」となっていました… https://github.com/ios-osushi/release-subscriptions/commit/22a6c517e999150f7a6fd9027989d56658c96358 で対応しました

Reply:
なるほどw
あれ、let 使えないっけ？

Reply:
ああぁっ…！！！！

Reply:
https://github.com/ios-osushi/release-subscriptions/commit/71d544eee0655719be0a2d4b11b7fa1d45f4ef2b で再修正しました 🙇

Reply:
LGTM :)